README.md

```diff
@@ -154,6 +154,34 @@ then 4 times in a row and issue an appropriate alert.
 
 ## Running a test environment
 
+### Kwok Container
+
```

Comment:
I think it's worth adding a sentence here about what is Kwok.

Reply:
Done